include/TPP/PassBundles.td

```diff
@@ -10,6 +10,7 @@
 #define TPP_DIALECT_TPP_PASSBUNDLES
 
 include "mlir/Pass/PassBase.td"
+include "mlir/Dialect/PDL/IR/PDLDialect.td"
```

Comment:
Is this needed? I expect you don't need any symbols from this file in `PassBundles.td`.

Reply:
You're right, removed.